src/ui/ResponsiveComponents/ResponsiveDropdown/ResponsiveDropdown.ts

```diff
@@ -80,6 +84,24 @@ export class ResponsiveDropdown {
       .build();
   }
 
+  private closeIfOpened = () => {
+    this.isOpened && this.close();
```

Comment:
Is it necessary to check if `isOpened` is true, or could we just call `this.close` regardless and remove this method?

Reply:
I prefer not calling it for no reason